.github/workflows/pull-request.yml

```diff
@@ -162,10 +162,8 @@ jobs:
             - name: Run NPM CI
               run: npm ci
 
-            - name: Download Local Bundle
-              uses: actions/download-artifact@v3
-              with:
-                  name: bundle-local
+            - name: Build Files
```

Comment:
I think each test should build its own files, rather than running the entire build. Would make it easier to troubleshoot in the future if a single task is off.